views/pulsar/v2/helpers/nav.html.twig

```diff
@@ -56,12 +56,12 @@
 {% endmacro %}
 
 
-{% macro nav_item(options) %}
+{% macro nav_item(options, index) %}
 {% spaceless %}
     {% import '@pulsar/pulsar/v2/helpers/html.html.twig' as html %}
     {% import '@pulsar/pulsar/v2/helpers/util.html.twig' as util %}
 
-    <li
+    <li id="nav-item-{{ index }}"
```

Comment:
There's two separate loops that use `nav_item` and I _think_ each one will kick off a new index, so straight away we'll have duplicate integers in those IDs like @mikedevelops suspected.

There's a more fundamental choice to be made first though... is the 'skip nav link' accessibility thing a CMS feature only? If so, I don't believe we should make this change here, the `nav_data` object that the CMS generates in it's page controller should be updated to pass an `id` attribute with their `$this->navigation` object, this will then be added to the nav item `li` by the attribute parser on L:66. That way, we're not generating redundant IDs for the other products.

More importantly, if anyone tries to pass an ID, or the controller is updated to do it, your change here on L:64 wouldn't be merged with the other ID and would now result in:

```
<li id="nav-item-0" id="foo">
```

**But**, if you plan on adding this skip nav link to Pulsar core, then we can go ahead with the change, but we'll need to find a way to:

* Avoid duplicated ID attributes (`id="a" id="b"`)
* Avoid duplicated ID values from `loop.index`
* Merge user-supplied IDs with the index ID

There's maybe something we could do with `loop.parent`, but that might only work with nested loops so don't quote me on that... first step would be adding some failing unit tests which we can work through tomorrow 😀